databases.md

```diff
@@ -0,0 +1,48 @@
+# Langley Foxall Databases
+
+## Platforms
+### Web
+Web applications will use [MySQL](https://www.mysql.com/) by default unless under exceptional circumstances and approved
+by a lead developer.
+
+Windows Stack applications will use MSSQL.
+
+### UWP (Offline) 
+UWP applications requiring offline functionality will use SQLite.
+
+### iOS
+iOS applications can either use CoreData OR Realm.
+
+### Android
+Android applications can use Realm.
+
+
+### Approved Database Management Tools
+Below are a list of approved database management tools that you can download and use.
+
+* [Sequel Pro (Mac)](https://www.sequelpro.com/)
+* [DataGrip (Universal)](https://www.jetbrains.com/datagrip/download/)
+* [MySQL Workbench (Universal)](https://www.mysql.com/products/workbench/)
+* [Realm Studio (Universal)](https://realm.io/products/realm-studio/)
+
+## Practices
+
+### Identitys
+
+ID's should be either auto-incrementing unsigned integers, or GUIDs/UUIDs. 
```

Comment:
Perhaps it's worth mentioning that in SQL you do this by creating an `IDENTITY` on a specific field